torch/csrc/jit/init.cpp

```diff
@@ -223,26 +223,25 @@ void initJITBindings(PyObject *module) {
         return createPyObjectForStack(std::move(stack));
       });
 
+      py::class_<PyTorchFileWriter>(m, "PyTorchFileWriter")
+        .def(py::init<std::string>())
+        .def("write_record", &PyTorchFileWriter::writeRecord)
+        .def("write_end_of_file", &PyTorchFileWriter::writeEndOfFile);
 
-    py::class_<PyTorchFileWriter>(m, "PyTorchFileWriter")
-      .def(py::init<std::string>())
-      .def("write_record", &PyTorchFileWriter::writeRecord)
-      .def("write_end_of_file", &PyTorchFileWriter::writeEndOfFile);
-
-    py::class_<PyTorchFileReader>(m, "PyTorchFileReader")
-      .def(py::init<std::string>())
-      .def("get_record_with_key", [](PyTorchFileReader &self, uint64_t key) {
-        at::DataPtr data;
-        size_t size;
-        std::tie(data, size) = self.getRecordWithKey(key);
-        return py::bytes(reinterpret_cast<const char*>(data.get()), size);
-      })
-      .def("get_last_record", [](PyTorchFileReader &self){
-        at::DataPtr data;
-        size_t size;
-        std::tie(data, size) = self.getLastRecord();
-        return py::bytes(reinterpret_cast<const char*>(data.get()), size);
-      });
+      py::class_<PyTorchFileReader>(m, "PyTorchFileReader")
+        .def(py::init<std::string>())
+        .def("get_record_with_key", [](PyTorchFileReader &self, uint64_t key) {
+          at::DataPtr data;
+          size_t size;
+          std::tie(data, size) = self.getRecordWithKey(key);
+          return py::bytes(reinterpret_cast<const char*>(data.get()), size);
+        })
+        .def("get_last_record", [](PyTorchFileReader &self){
+          at::DataPtr data;
+          size_t size;
+          std::tie(data, size) = self.getLastRecord();
+          return py::bytes(reinterpret_cast<const char*>(data.get()), size);
+        });
```

Comment:
I'm confused. What has changed here?

Reply:
Nothing, it probably just got indented differently. Reverting

Reply:
Nah it's ok. You can leave it. It was just confusing